### PR TITLE
Added Label

### PR DESCRIPTION
**Pull request by sainath7878**
Files to check:

- [src/components/label/Label.jsx](https://github.com/sainath7878/joddkeep/compare/add-label?expand=1#diff-305cb171f36c9e899df8cc12eea2a4e1b2466f62e02c8aac74346f11675c9a2d)
- [src/pages/labelPage/LabelPage.jsx](https://github.com/sainath7878/joddkeep/pull/9/files#diff-c4494edc61c946d4ba71622361e0c5ed5870e9d442b972e19b994be330f6d0bc)
- [src/pages/labelPage/labelPage.css](https://github.com/sainath7878/joddkeep/pull/9/files#diff-1903bf3693adee1b070209c3806fd17568146f4d421d7bb147d5f6a48db940e3)
- [src/pages/notFound/NotFound.jsx](https://github.com/sainath7878/joddkeep/pull/9/files#diff-de85933229bff783692e90bd7453bf4c803955e6858946ec8f9316e352a1da0a)
- [src/pages/notFound/notFound.css](https://github.com/sainath7878/joddkeep/pull/9/files#diff-4747fb2861fd45cc6ab0bac2a78be5927889fc23126b1dfefc99ff46da9503a9)

### CODE REVIEW

File: src/pages/labelPage/LabelPage.jsx [hunk 1]
```diff
@@ -1,9 +1,202 @@
 import { Label } from "components/index";
+import { useDocument } from "customHooks/useDocument";
+import "./labelPage.css";
+import { useEffect, useState } from "react";
+import { useNotes } from "context";
 
 function LabelPage() {
+  useDocument("Label");
```
Comment (anik31) [inline src/pages/labelPage/LabelPage.jsx:8]: appreciation - liked that you're using an useDocument hook

File: src/pages/labelPage/LabelPage.jsx [hunk 2]
```diff
@@ -1,9 +1,202 @@
...
+      newData.sort((a, b) => getTime(a.createdDate) - getTime(b.createdDate));
+    }
+    if (filters.all) {
+      setFilteredData(newData);
+    } else {
+      const selectedLabels = labels.filter((item) => filters[item]);
+      newData = newData.filter((item) => selectedLabels.includes(item.label));
+      setFilteredData(newData);
+    }
+  }, [filters]);
```
Comment (anik31) [inline src/pages/labelPage/LabelPage.jsx:56]: appreciation - good that you're doing the filteration only if filter state changes

File: src/pages/labelPage/LabelPage.jsx [hunk 3]
```diff
@@ -1,9 +1,202 @@
...
+              <input
+                type="radio"
+                className="form-radio"
+                name="sort"
+                value="low-to-high"
+                onChange={() =>
+                  setFilters({
+                    ...filters,
+                    lowToHigh: !filters.lowToHigh,
+                    highToLow: false,
```
Comment (anik31) [inline src/pages/labelPage/LabelPage.jsx]: query - is the price filter required here ?

File: src/pages/notFound/NotFound.jsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import { Link } from "react-router-dom";
+
+function NotFound() {
+  return (
+    <div className="not-found-container">
+      <img src={oops} alt="oops" height="500px" />
+      <Link to="/">
+        <button className="btn btn-secondary fs-s">Go back to Homepage</button>
+      </Link>
+    </div>
```
Comment (anik31) [inline src/pages/notFound/NotFound.jsx]: appreciation - good that you've added 404 page

**Review by anik31 — APPROVED** [review]
Everything looks good. Code is clean & UI is also good.

File: src/pages/labelPage/LabelPage.jsx (hunk 3 above)
Comment (sainath7878) [reply src/pages/labelPage/LabelPage.jsx]: Forgot to remove. Thanks for pointing!

**Comment by Dev-AkshatGupta** [comment]
Everything seems really nothing special to point just remove some unnecessary space else everything is just good to merge

File: src/components/displayNote/displayNote.css [hunk 5]
```diff
@@ -16,6 +16,12 @@
   word-wrap: break-word;
 }
 
+.display-date {
+  /* color: #a8a29e; */
```
Comment (Maianki) [inline src/components/displayNote/displayNote.css:20]: Please remove commented code.

File: src/components/newNote/NewNote.jsx [hunk 6]
```diff
@@ -5,6 +5,7 @@ import { ColorPalette } from "components/index";
 import { useNotes } from "context";
 
 function NewNote() {
+  // const d = new Date();
```
Comment (Maianki) [inline src/components/newNote/NewNote.jsx:8]: Here also you can remove this unwanted code.

File: src/customHooks/useDocument.jsx [hunk 7]
```diff
@@ -0,0 +1,5 @@
+const useDocument = (title) => {
+  document.title = title;
+};
+
+export { useDocument };
```
Comment (Maianki) [inline src/customHooks/useDocument.jsx:5]: apppreciation - custom hook for page title

**Review by Maianki — COMMENTED** [review]
Good job Sainath. You can go ahead with merging nothing much to suggest just remove the commented code.

**Pull request merged**